tests/test_vcap_services.py

```diff
@@ -24,18 +24,42 @@ class TestVcapServices(unittest.TestCase):
     @patch.dict(os.environ, {'VCAP_SERVICES': json.dumps(VCAP_SERVICES)})
     def test_get_database_uri_vcap(self):
         """ Test if it gets the vcap db uri """
-        db_uri = 'mysql+pymysql://{}:{}@{}:{}/{}'.format(
-            'vcap_user', 'vcap_pass', '127.0.0.2', '3307', 'vcap')
+        if 'VCAP_SERVICES' in os.environ:
+            DATABASE_URI = os.getenv('DATABASE_URI', 'mysql+pymysql://vcap_user:vcap_pass@127.0.0.2:3307/vcap')
+        elif 'TRAVIS' in os.environ:
+            DATABASE_URI = os.getenv('DATABASE_URI', 'mysql+pymysql://root:@localhost:3306/shopcarts')
+        else:
+            DATABASE_URI = os.getenv('DATABASE_URI', 'mysql+pymysql://root:root@localhost:3306/shopcarts')
 
+        db_uri = DATABASE_URI
```

Comment:
Don't need the big `if/elif/else` in every test. Just set `db_uri` for the one it should be (vcap, travis, local) in that test.